src/dialog/dialog.directive.ts

```diff
@@ -61,7 +61,7 @@ export class DialogDirective implements OnInit, OnDestroy, OnChanges {
 	 * @type {string}
 	 * @memberof DialogDirective
 	 */
-	@Input() wrapperClass: string;
+	@Input() wrapperClass = "";
```

Comment:
It can't be undefined or empty string signifies something?

Reply:
It could probably go back to undefined ... I tend to prefer initializing to useful (non undefined) values where possible though.

Reply:
That implies that a value is needed when it's not.

Reply:
Not really? It just implies that it's set to an empty string